docs/user-guides/cluster/oidc_connectivity.md

```diff
@@ -0,0 +1,179 @@
+```
+
+With the combination of _Service Account Issuer Discovery_ and _Structured Authentication Configuration_, Cluster to
+Cluster trust can be established.
+
+The remote clusters can add the `Greenhouse` cluster's _Service Account Issuer_ as an
+OIDC issuer in their _Structured Authentication Configuration_ thus allowing `Greenhouse` cluster to authenticate
+against
+remote cluster, using its in-cluster service account token.
+
+The **OIDC Remote Cluster Connectivity** is illustrated below -
+
+![OIDC Remote Connectivity](../oidc-remote-connectivity.png)
+
+### Preparation
+
+The `greenhouse` cluster should expose the `/.well-known/openid-configuration` over an unauthenticated endpoint to allow
+remote clusters to fetch the OIDC configuration.
+
+```NOTE
+Some cloud providers or managed Kubernetes services might not expose this as an unauthenticated endpoint. In such
+cases, you can serve this configuration from a different endpoint and set this as the discoveryURL 
+in structured authentication configuration.
+
+check out https://kubernetes.io/docs/reference/access-authn-authz/authentication/#using-authentication-configuration
+for more information 
+```
+
+Configure the OIDC issuer in the `Structured Authentication Configuration` of the remote cluster
+
+**Example Structured Authentication Configuration file**
+
+```yaml
+apiVersion: apiserver.config.k8s.io/v1beta1
+kind: AuthenticationConfiguration
+jwt:
+  - issuer:
+      url: https://<greenhouse-service-account-issuer>
+      audiences:
+        - greenhouse # audience should be greenhouse
+    claimMappings:
+      username:
+        claim: 'sub' # claim to be used as username
+        prefix: 'greenhouse:' # prefix to be added to the username to prevent impersonation (can be any string of your choice)
+# additional trusted issuers
+#     - issuer:
+```
+
+Add RBAC rules to the remote cluster to allow the `Greenhouse` cluster to be authorized to CRUD kubernetes resources
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: greenhouse-<cluster-name>-oidc-access
+subjects:
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: greenhouse:system:serviceaccount:<your-organization-namespace>:<cluster-name>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+```
+
+The subject kind `User` name must follow the pattern of
+`<prefix>:system:serviceaccount:<your-organization-namespace>:<cluster-name>`
+
+`<prefix>` is the prefix used in the `Structured Authentication Configuration` file for the `username` claim mapping
+
+```NOTE
+For convenience purposes, the `prefix` is set to `greenhouse:` in the example `Structured Authentication Configuration`
+but it can be any string identifier of your choice
+
+if you use '-' in prefix, for example, `identifier-` then the subject name should be `identifier-system:serviceaccount:<your-organization-namespace>:<cluster-name>`
```

Comment:
```suggestion
For convenience purposes, the `prefix` is set to `greenhouse:` in the example `Structured Authentication Configuration`
but it can be any string identifier of your choice.

If you use '-' in prefix, for example, `identifier-` then the subject name should be `identifier-system:serviceaccount:<your-organization-namespace>:<cluster-name>`.
```